components/flex/style/index.less

```diff
@@ -0,0 +1,127 @@
+@import '../../style/themes/index';
+@import '../../style/mixins/index';
+
+@flex-prefix-cls: ~'@{ant-prefix}-flex';
+
+.@{flex-prefix-cls} {
+  display: flex;
+  gap: var(--flex-gap, 0);
+  flex: var(--flex, unset);
+
+  &-direction {
+    &-row {
+      flex-direction: row;
+    }
+
+    &-column {
+      flex-direction: column;
+    }
+
+    &-row-reverse {
+      flex-direction: row-reverse;
+    }
+
+    &-column-reverse {
+      flex-direction: column-reverse;
+    }
+  }
+
+  &-justify {
+    &-flex-start {
+      justify-content: flex-start;
+    }
+
+    &-center {
+      justify-content: center;
+    }
+
+    &-flex-end {
+      justify-content: flex-end;
+    }
+
+    &-space-between {
+      justify-content: space-between;
+    }
+
+    &-space-around {
+      justify-content: space-around;
+    }
+
+    &-space-evenly {
+      justify-content: space-evenly;
+    }
+
+    &-start {
+      justify-content: start;
+    }
+
+    &-end {
+      justify-content: end;
+    }
+
+    &-right {
+      justify-content: right;
+    }
+
+    &-left {
+      justify-content: left;
+    }
+
+    &-stretch {
+      justify-content: stretch;
+    }
+
+    &-normal {
+      justify-content: normal;
+    }
+  }
+
+  &-align {
+    &-flex-start {
+      align-items: flex-start;
+    }
+
+    &-center {
+      align-items: center;
+    }
+
+    &-flex-end {
+      align-items: flex-end;
+    }
+
+    &-start {
+      align-items: start;
+    }
+
+    &-end {
+      align-items: end;
+    }
+
+    &-stretch {
+      align-items: stretch;
+    }
+
+    &-normal {
+      align-items: normal;
+    }
+  }
+
+  &-wrap {
+    &-wrap {
+      flex-wrap: wrap;
+    }
+
+    &-nowrap {
+      flex-wrap: nowrap;
+    }
+
+    &-wrap-reverse {
+      flex-wrap: wrap-reverse;
+    }
+  }
+
+  &-center {
+    align-items: center !important;
+    justify-content: center !important;
+  }
```

Comment:
We don't need them because ant-design doesn't have them. We want to be consistent with ant-design as much as possible.

Reply:
I appreciate your input. I will fully adhere to the ant-design standards in my future work.

Reply:
So let's delete it :)